pallets/rmrk-equip/src/tests.rs

```diff
@@ -50,33 +50,53 @@ fn create_base_works() {
 			Origin::signed(ALICE), // origin
 			bvec![0u8; 20],        // base_type
 			bvec![0u8; 20],        // symbol
-			vec![PartType::FixedPart(fixed_part), PartType::SlotPart(slot_part),],
+			bvec![PartType::FixedPart(fixed_part), PartType::SlotPart(slot_part),],
 		));
 
 		// println!("{:?}", RmrkEquip::bases(0).unwrap());
 	});
 }
 
-/// Base: Attempting to create a base with more the max parts fails
-#[test]
-fn exceeding_max_parts_per_base_fails() {
-	ExtBuilder::default().build().execute_with(|| {
-		let mut parts = Vec::<PartType<BoundedVec<u8, UniquesStringLimit>>>::new();
-		for i in 100..110 {
-			let fixed_part = FixedPart { id: i, z: 0, src: stb("fixed_part_src") };
-			parts.push(PartType::FixedPart(fixed_part));
-		}
+// no longer needed as we use PartsLimit on BoundedVec
```

Comment:
good to remove if not needed.

Reply:
[196c9d8](https://github.com/rmrk-team/rmrk-substrate/pull/95/commits/196c9d8165e37bc0e2c93d132f456a0f049ea5f4)